aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
+   
+   ```diff
+   - @page "/counter"
+   + @page "/counter-2"
+   ```
+     
+   Also, change the interactive render mode using ***either*** of the following lines:
+   
+   ```razor
+   @attribute [RenderModeInteractiveServer]
+   ```
+
+   <!-- UPDATE 8.0 To adopt simplified naming at RTM -->
+
+   ```razor
+   @rendermode RenderMode.InteractiveServer
+   ```
+
+   For Blazor Server components that should also strictly adopt the interactive server render mode, add ***either*** of the preceding directives to them and leave them in the Blazor Server project's `Pages`/`Shared` folders. Such components only ever render on the server. Component code is kept private on the server using the interactive server render mode.
+
+   When the app is run in the next step, navigate to each of the counter components (`/counter` and `/counter-2`) to inspect how they render.
+   
+   If any of your server-side components should also disable prerendering, see <xref:blazor/components/render-modes#prerendering>.
+
+   For any components that should strictly adopt the interactive WebAssembly render mode in the `.Client` project (`Pages` folder of the `Net8BlazorServer.Client` project), explicitly configure the render mode:
+
+   ```razor
+   @attribute [RenderModeInteractiveWebAssembly]
+   ```
+
+   <!-- UPDATE 8.0 To adopt simplified naming at RTM -->
+
+   ```razor
+   @rendermode RenderMode.InteractiveWebAssembly
+   ```
+
+   Such components only render on the client after the Blazor bundle has been downloaded and the Blazor runtime activates. Keep in mind that component code is ***not*** private using the interactive WebAssembly render mode.
+
+   For additional information on applying render modes to components, see <xref:blazor/components/render-modes>.
+
+1. Run the solution from the ***Blazor Server*** project:
+
+   For Visual Studio, confirm that the Blazor Server project is selected in **Solution Explorer** when running the app.
+   
+   If using the .NET CLI, run the project from the Blazor Server project's folder.
+
+   For the `Counter` component of the `.Client` project, navigate to `/counter`.
+
+   If you added the `Counter2` component, which is strictly configured for interactive server rendering, navigate to `/counter-2`.
+
+### Convert a Blazor Server app into a Blazor Web App
+
+To adopt all of the changes that were made to the project templates for this release, we recommend the following process:
+
+* Create a new project from the Blazor Web App project template. For more information, see <xref:blazor/tooling>.
+* Move the app's components and code to the new app, making modifications to adopt new features.
+* Update the layout and styles of the Blazor Web App to match the Blazor Server app.
+
+New .NET 8 features are covered in <xref:aspnetcore-8#blazor>. When updating an app from .NET 6 or earlier, see the migration and release notes (*What's new* articles) for intervening releases.
+
```

Comment:
I'm proposing this is covered in the new Adopt Blazor Web App conventions section.

```suggestion
```